deploy/packages/serial-libraries.yaml

```diff
@@ -43,3 +43,12 @@ packages:
     specs:
       - python@2.7.15
       - python@3.6.5
+
+  gnu-stable-serial:
+    target_matrix:
+      - gnu-stable
+    requires:
+      - architecture
+      - compiler
+    specs:
+      - boost
```

Comment:
As this is common requirement from many packages, moving this to `gnu-intel-stable-serial` make sense?

Reply:
This is just a database registration of an external package for module generation. Other stages can register it as `%intel` package.

Reply:
Ahh..you are right! boost is in external-libraries already.